src/hotspot/share/nmt/memMapPrinter.cpp

```diff
@@ -27,6 +27,7 @@
 
 #ifdef LINUX
 
+#include "code/codeCache.hpp"
```

Comment:
The include lines below are weirdly sorted. Could you fix it while you are changing this file?